### PR TITLE
Add Chatbot arena

### PR DESCRIPTION
**Pull request by the-infiltrator**
image

All checks pass. Dataset added with a login since its gated on HF.

### CODE REVIEW

**Review by shayne-longpre — APPROVED** [review]
Minor changes then approved to merge! Thanks

File: data_summaries/ChatbotArena.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        "Task Categories": [
+            "Natural Language Processing Question Answering",
+            "Alignment",
+            "RLHF",
+            "Response Ranking",
+            "Text Quality Evaluation",
+            "Long-form Response Evaluation"
+        ],
+        "Text Sources": [
+            "https://huggingface.co/datasets/lmsys/chatbot_arena_conversations"
```
[the remark this answers is not in the document]
Comment (the-infiltrator) [reply data_summaries/ChatbotArena.json:26]: Resolved

File: data_summaries/Gorilla.json [hunk 2]
```diff
@@ -4,9 +4,9 @@
         "Dataset Name": "gorilla-apibench",
         "Collection": "Gorilla",
         "Collection URL": "https://gorilla.cs.berkeley.edu/",
-        "Dataset URL": "https://gorilla.cs.berkeley.edu/",
+        "Dataset URL": "https://huggingface.co/datasets/gorilla-llm/Berkeley-Function-Calling-Leaderboard",
```
[the remark this answers is not in the document]
Comment (the-infiltrator) [reply data_summaries/Gorilla.json:7]: Reverted back to original

File: data_summaries/ChatbotArena.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        ],
+        "Licenses": [
+            {
+                "License": "CC BY 4.0",
+                "License URL": "https://creativecommons.org/licenses/by/4.0/"
+            },
+            {
+                "License": "CC BY-NC 4.0",
+                "License URL": "https://creativecommons.org/licenses/by-nc/4.0/"
+            }
```
[the remark this answers is not in the document]
Comment (the-infiltrator) [reply data_summaries/ChatbotArena.json:59]: Added openai, need help populating other licenses for this dataset

**Pull request merged**